src/node_file.h

```diff
@@ -79,12 +86,30 @@ class BindingData : public SnapshotableObject {
   SERIALIZABLE_OBJECT_METHODS()
   SET_BINDING_ID(fs_binding_data)
 
+  static void LegacyMainResolve(
+      const v8::FunctionCallbackInfo<v8::Value>& args);
+
+  static void CreatePerIsolateProperties(IsolateData* isolate_data,
+                                         v8::Local<v8::ObjectTemplate> ctor);
+  static void RegisterExternalReferences(ExternalReferenceRegistry* registry);
+
   void MemoryInfo(MemoryTracker* tracker) const override;
   SET_SELF_SIZE(BindingData)
   SET_MEMORY_INFO_NAME(BindingData)
 
  private:
   InternalFieldInfo* internal_field_info_ = nullptr;
+
+  static FilePathIsFileReturnType FilePathIsFile(Environment* env,
+                                                 const std::string& file_path);
+
+  static const std::array<std::string, 10> legacy_main_extensions;
```

Comment:
```suggestion
  constexpr std::array<std::string, 10> legacy_main_extensions = {...};
```

Reply:
Do not compile: https://stackoverflow.com/a/71882832

```
In file included from ../../src/dataqueue/queue.h:9,
                 from ../../test/cctest/test_dataqueue.cc:1:
../../src/node_file.h:110:3: error: non-static data member ‘legacy_main_extensions’ declared ‘constexpr’
  110 |   constexpr std::array<std::string, 10> legacy_main_extensions = {
      |   ^~~~~~~~~
[4/140] CXX obj/src/dataqueue/libnode.queue.o
```